app/models/exception_hunter/error.rb

```diff
@@ -7,6 +7,11 @@ class Error < ApplicationRecord
 
     before_validation :set_occurred_at, on: :create
 
+    def add_user(user)
+      self.user_data = UserAttributesCollector.collect_attributes(user)
+      save!
+    end
```

Comment:
This is something the error creator should do. Consider the case where we move to ES, this would have no effect and we would have to repeat the logic. Plus it's not something the model should do, it falls more under the category of sanitizing data 🧼 .